pkg/pillar/devicenetwork/devicenetwork.go

```diff
@@ -75,17 +76,27 @@ func IsProxyConfigEmpty(proxyConfig types.ProxyConfig) bool {
 	return false
 }
 
-// Check if device can talk to outside world via atleast one of the free uplinks
+// VerifyDeviceNetworkStatus
+//  Check if device can talk to outside world via atleast one of the
+//  free uplinks
+// Return Values:
+//  Success / Failure
+//  error - Overall Error
+//  map[string]error - Map of interfaces with error Key: ifname.
```

Comment:
Should there be a period before "Key". I read it as there being some "error key".

Reply:
Done.. Fixed..